openstudiocore/src/model/FanConstantVolume.cpp

```diff
@@ -481,10 +483,15 @@ namespace detail {
     return types;
   }
 
-  std::string HVACComponent_Impl::emsActuatorComponentType() const {
+  std::string FanConstantVolume_Impl::emsActuatorComponentType() const {
     return std::string("FAN");
   }
 
+  std::vector<std::string> FanConstantVolume_Impl::emsInternalVariables() const {
+    std::vector<std::string> types{"Fan Maximum Mass Flow Rate", "Fan Nominal Pressure Rise", "Fan Nominal Total Efficiency"};
```

Comment:
Might consider using these fancy initializer lists instead of push_back. I think it is much faster because it doesn't require reallocation every time you push.

Reply:
oh I see. for output variables, you just followed the code that was there... sorry. no need to update those existing ones.

Reply:
Just randomly came accross this. I added most of the output variables for all objects, but I just followed the existing examples with push_back. If we want to replace with an initializer this could be scripted.
Note that currently the overhead wouldn't be too huge because it's static...
Also note that it should ideally be refactored  to not be static... because in some objects the output variables are dependent on the choices you made (eg, [WaterHeaterMixed](https://github.com/jmarrec/OpenStudio/blob/develop/openstudiocore/src/model/WaterHeaterMixed.cpp#L117))

Reply:
@jmarrec are your output variables pushed to develop or in a branch still?

Reply:
@jmarrec - @kbenne, @brianlball, and I had an extensive conversation about whether or not our output variables should be static or not.  In the end, the conclusion was that we should standardize on the outputVariableNames method being static and returning all possible output variable names.  If we want to make a dynamic method that could be achieved by creating a new method or by adding an argument to the outputVariableNames method.  I'll try to remember/summarize the logic in our decision but will probably forget some things:

- Making the call dynamic would imply a level of logic and support for output variables and EMS that we currently do not have (e.g. checking them for correctness at translation time), the current approach is to offer a super set to the user and all failures at E+ run time
- Keeping this method static allows us to quickly get all possible output variables without having to query all objects in the model which would be time consuming